plexapi/video.py

```diff
@@ -84,6 +84,24 @@ def markUnwatched(self):
         self._server.query(key)
         self.reload()
 
+    def augmentation(self):
+        """ Returns a list of :class:`~plexapi.library.Hub` objects.
+
+            augmentation returns hub items relating to online media sources
+            such as Tidal Music "Track From {item}" or "Soundtrack of {item}"
+            Plex Pass and linked Tidal account are required
+
+        """
+        account = self._server.myPlexAccount()
+        tidalOptOut = [service.value for service in account.onlineMediaSources()
+                       if service.key.endswith('music')][0]
+        if account.subscriptionStatus != 'Active' or tidalOptOut == 'opt_out':
+            raise BadRequest('Requires Plex Pass and Tidal Music enabled.')
+        data = self._server.query(self.key + '?asyncAugmentMetadata=1')
+        mediaContainer = MediaContainer(data=data, server=self._server)
+        augmentationKey = mediaContainer.augmentationKey
```

Comment:
Do we need `MediaContainer`?

```python
augmentationKey = data.attrib.get('augmentationKey')
```

Reply:
You can keep the `MediaContainer` class. It might come in useful later. In this case you're creating a `mediaContainer` instance, then just throwing it away after getting the `augmentationKey`.